admin/admin-model.php

```diff
@@ -288,33 +295,69 @@ public function update_language( $args ) {
 	 * @since 3.4
 	 *
 	 * @param array $args {
-	 *   @type int    $lang_id        Id of the language to modify.
-	 *   @type string $name           Language name ( used only for display ).
-	 *   @type string $slug           Language code ( ideally 2-letters ISO 639-1 language code ).
-	 *   @type string $locale         WordPress locale. If something wrong is used for the locale, the .mo files will not be loaded...
-	 *   @type int    $rtl            1 if rtl language, 0 otherwise.
-	 *   @type int    $term_group     Language order when displayed.
-	 *   @type string $flag           Optional, country code, @see flags.php.
+	 *   @type string $name       Language name (used only for display).
+	 *   @type string $slug       Language code (ideally 2-letters ISO 639-1 language code).
+	 *   @type string $locale     WordPress locale. If something wrong is used for the locale, the .mo files will not be
+	 *                            loaded...
+	 *   @type int    $rtl        1 if rtl language, 0 otherwise.
+	 *   @type int    $term_group Language order when displayed.
+	 *   @type int    $lang_id    Optional, ID of the language to modify. An empty value means the language is being
+	 *                            created.
+	 *   @type string $flag       Optional, country code, {@see settings/flags.php}.
 	 * }
-	 * @return mixed The serialized description array updated.
+	 * @return string The serialized description array updated.
 	 */
-	protected function update_language_description( $args ) {
-		$language_term = get_term( (int) $args['lang_id'] );
+	protected function update_language_description( array $args ) {
```

Comment:
I like the name of your filter. Maybe we could rename this method the same way.

Reply:
Renamed into `build_language_metas()` in [f3c72a8](https://github.com/polylang/polylang/pull/1201/commits/f3c72a86f125fdcfe952183a88f44803d6e6d8f2). I thought about `build_and_serialize_language_metas()` but it's a bit long 😅